Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM debian:bullseye
 # # (for example docker build --build-arg SPACK_VERSION=v0.16.2)
 ARG SPACK_VERSION=develop
 ARG OCT_VERSION=13.0
+ARG BER_VER=""
```

Comment:
I see the advantage of a short name (`BER`), but on reflection I think I prefer a longer name (`BERKELEYGW`) so that in particular in the Makefile it is clear what the argument does when we call docker.